code-challenges.test.js

```diff
@@ -35,26 +75,90 @@ const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
 // b) Create the function that makes the test pass.
+        
+        // Pseudo code:
+                
+                 // Going to creat a function named objDivBy3 and will take an object as an argument 
+                // Going to write if statement to check if given number is divisible by 3 
+                 //Going to use string interpolation to keep the function dynamic with given value
+
+const objDivBy3 = (obj) => {
+    if (obj.number % 3 == 0){
+        return `${obj.number} is divisible by three`
+    } else if (obj.number % 3 !== 0) {
+        return`${obj.number} is not divisible by three`
+    }
+}
 
-// Pseudo code:
+
+    
+  
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+describe("capsOnAll", () => {
+    it("returns an array with all the words capitalized", () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+        // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+        // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+      expect(capsOnAll(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+      expect(capsOnAll(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+     })
+  })
+            //Problem 2 fist test - ReferenceError: capsOnAll is not defined
+            //Problem 2 passed second test 
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+        // Pseudo code:
+                // Create a function called capsOnAll that takes in an array as an argument
+                //Going to use .map() to iterate over array
+                //Going to make two variable, one to separate first letter from rest of the word, by using charAt(0) to target first letter, then .toUpperCase()to get capital letter
+                //Going to slice the word from second character to the rest of the word by using .splice, and lower casing the rest by using .toLowerCase
```

Comment:
Great pseudo code again. Quick note, you said .splice here when in the code you used .slice.